tests/complete_run.py

```diff
@@ -2,7 +2,7 @@
 import unittest
 
 # This test should be run with the latest E3SM Diags tutorial code.
-from examples.tutorials.tutorial_v2_4_0_all_sets_E3SM_machines import run_lcrc
+from examples.run_v2_5_0_all_sets_E3SM_machines import run_lcrc
```

Comment:
As mentioned above, maybe we import a copy of the file that uses `e3sm_diags` instead of `acme_diags`?